mappings/17w13b#17w14a.tinydiff

```diff
@@ -714,13 +714,13 @@ c	net/minecraft/unmapped/C_9590849
 	m	(Lnet/minecraft/unmapped/C_0539808;)Ljava/util/Collection;	m_7892993	m_7892993	
 		p	1		p_1	
 c	net/minecraft/unmapped/C_9639106
-	m	(II)V	m_1779891		m_1779891
-		p	1			p_1
-		p	2			p_2
+	m	(II)V	m_1779891		conditionalRenderToolTip
```

Comment:
is a `renderTooltipIf<condition>` feasible here?

Reply:
It just checks if you are not dragging a stack and that there is a stack in the slot you are hovering.
So there are no additional conditions than you expect from renderToolTip.
Iirc I named it like that bc I thought I couldn't use renderToolTip bc that method name is already used (but it is with diff args so I can actually use it or at least enigma allows it)